react-d3-chart/src/components/Victory/LineChart/VictoryLineChart.stories.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import LineChart from '.';
+import { LineProps } from '../../types';
+import { graphData } from '../../../utils/rawData';
+
+const ranges = '1-5, 7-9';
+let data: Array<LineProps> = [];
+
+ranges.split(',').forEach((range) => {
+  const l = parseInt(range.trim().split('-')[0]);
+  const r = parseInt(range.trim().split('-')[1]) + 1;
+
+  for (let i = l; i < r; i++) {
+    const temp = graphData.trace[i].c3.map((value, index) => ({
+      x: index,
+      y: value,
+    }));
+    data.push([...temp]);
```

Comment:
data is an array of arrays? In my mind it should be [{x:1, y:2}, {x:2, y:3}, ...]